netty/src/main/java/io/grpc/netty/InternalNettyChannelBuilder.java

```diff
@@ -75,4 +77,25 @@ public static void setStatsRecordStartedRpcs(NettyChannelBuilder builder, boolea
   }
 
   private InternalNettyChannelBuilder() {}
+
+  /**
+   * Accessor for tests.
+   */
+  @VisibleForTesting
+  public static final class TestAccessor {
+    private TestAccessor() {}
+
+    /**
+     * Gets the {@link TransportCreationParamsFilterFactory} that is dynamically set to the
+     * specified NettyChannelBuilder at the moment.
+     */
+    @Nullable
+    public static TransportCreationParamsFilterFactory getDynamicTransportParamsFactory(
+        NettyChannelBuilder builder) {
+      if (builder.dynamicParamsFactory instanceof TransportCreationParamsFilterFactory) {
+        return (TransportCreationParamsFilterFactory) builder.dynamicParamsFactory;
```

Comment:
I would have to be convinced of the usefulness.  It only works if the tcpff was also set with this class, since it can only return the type from this class.  

Can you show some example usage of this class?

Reply:
User can only set tcpff by `InternalNettyChannelBuilder` class. I cc'ed you a CL the builder in which is hard to test without an accessor.

Reply:
Any insight making test easier?